### ISSUE TITLE
AndroidLibrary cannot run

### ISSUE THREAD

**Issue by dalewking** [lovelysystems/robotframework-androidlibrary#64]
The first problem is that calabash-android has moved on and is now at 0.4.x and no longer has an extract-manifest command. That causes the ExPat error reported in #63.

I tried backing up to an older version of calabash-android (0.3.x versions) which gets you past the extract-manifest itself, but then calabash fails to build the instrumentation because the old versions of calabash still expect aapt to be in the platform-tools directory and it was moved to a build-tools directory with later version of the Android tools.

So basically was unable to get this library to work.
